src/EventBuilder/EventBuilder.csproj

```diff
@@ -18,9 +18,14 @@
     <PackageReference Include="CommandLineParser" Version="1.9.71" />
     <PackageReference Include="Microsoft.Web.Xdt" Version="2.1.1" />
     <PackageReference Include="Mono.Cecil" Version="0.9.6.1" />
-    <PackageReference Include="NuGet.Core" Version="2.10.1" />
     <PackageReference Include="Nustache" Version="1.15.3.7" />
     <PackageReference Include="Polly" Version="3.0.0" />
     <PackageReference Include="Serilog" Version="1.5.14" />
+    <PackageReference Include="NuGet.Core" Version="2.14.0" />
   </ItemGroup>
+<ItemGroup>
```

Comment:
This formatting seems a bit inconsistent with the others, e.g. indent off by one